src/uct/sm/mm/mm_posix.c

```diff
@@ -17,12 +17,16 @@
 #define UCT_MM_POSIX_MMAP_PROT      (PROT_READ | PROT_WRITE)
 #define UCT_MM_POSIX_HUGETLB        UCS_BIT(0)
 #define UCT_MM_POSIX_SHM_OPEN       UCS_BIT(1)
-#define UCT_MM_POSIX_CTRL_BITS      2
+#define UCT_MM_POSIX_PROC_LINK      UCS_BIT(2)
+#define UCT_MM_POSIX_CTRL_BITS      3
+#define UCT_MM_POSIX_FD_BITS        29
```

Comment:
Thanks.
In this case, the orig_fd (in the attach function) may not be the same as the shm_fd (in the alloc function)? It will be the one packed to the mmid but not the original one from when the file was opened, right?
If so,  then we are relying on those 3 (lost) bits to be zeros anyway for it work?